### PR TITLE
Changed: Update dotnet core and ASP.NET core to 2.1

### PR DESCRIPTION
**Pull request by danielunderwood**

### CODE REVIEW

File: LidarrAPI/LidarrAPI.csproj [hunk 1]
```diff
@@ -1,23 +1,19 @@
...
-    <PackageReference Include="Pomelo.EntityFrameworkCore.MySql" Version="2.0.1"/>
-    <PackageReference Include="TraktApiSharp" Version="1.0.0-alpha2"/>
-    <PackageReference Include="SharpRaven" Version="2.3.2"/>
-  </ItemGroup>
-  <ItemGroup>
-    <DotNetCliToolReference Include="Microsoft.VisualStudio.Web.CodeGeneration.Tools" Version="2.0.0"/>
-    <DotNetCliToolReference Include="Microsoft.EntityFrameworkCore.Tools.DotNet" Version="2.0.0"/>
+    <PackageReference Include="DogStatsD-CSharp-Client" Version="3.1.0" />
+    <PackageReference Include="Microsoft.AspNetCore.App" />
+    <PackageReference Include="NLog" Version="5.0.0-beta11" />
```
Comment (Qstick) [inline LidarrAPI/LidarrAPI.csproj:11]: Probably for another PR, but we should bump to stable on NLog

File: LidarrAPI/LidarrAPI.csproj [hunk 2]
```diff
@@ -1,23 +1,19 @@
...
-    <PackageReference Include="Octokit" Version="0.29.0"/>
-    <PackageReference Include="Pomelo.EntityFrameworkCore.MySql" Version="2.0.1"/>
-    <PackageReference Include="TraktApiSharp" Version="1.0.0-alpha2"/>
-    <PackageReference Include="SharpRaven" Version="2.3.2"/>
-  </ItemGroup>
-  <ItemGroup>
-    <DotNetCliToolReference Include="Microsoft.VisualStudio.Web.CodeGeneration.Tools" Version="2.0.0"/>
-    <DotNetCliToolReference Include="Microsoft.EntityFrameworkCore.Tools.DotNet" Version="2.0.0"/>
+    <PackageReference Include="DogStatsD-CSharp-Client" Version="3.1.0" />
+    <PackageReference Include="Microsoft.AspNetCore.App" />
```
Comment (Qstick) [inline LidarrAPI/LidarrAPI.csproj:10]: Used to be AspNetCore.All? Name change on package? Also do we need to explicit call a version here?

File: LidarrAPI/LidarrAPI.csproj (hunk 1 above)
Comment (Qstick) [reply LidarrAPI/LidarrAPI.csproj:11]: From Nlog git: With the release of NLog 4.5, NLog 5 is now obsolete and will be unlisted soon from nuget. 

Craziness but seems 5 beta has been replaced by 4.5 lol

File: LidarrAPI/LidarrAPI.csproj (hunk 2 above)
Comment (danielunderwood) [reply LidarrAPI/LidarrAPI.csproj:10]: Yeah the package name changed and I think the dependencies it brings in did as well. The docs I saw recommended no version, but I don't know a ton about packages in .NET. You can see the second and third bullets [here](https://docs.microsoft.com/en-us/aspnet/core/migration/20_21?view=aspnetcore-2.1).

**Pull request merged**